cmd/controller/certificates.go

```diff
@@ -189,10 +190,14 @@ func (c *Certificates) Init(ctx context.Context) error {
 		"kubernetes.default.svc",
 		"kubernetes.default.svc.cluster",
 		fmt.Sprintf("kubernetes.svc.%s", c.ClusterSpec.Network.ClusterDomain),
-		"127.0.0.1",
```

Comment:
Might be worthwhile to include 127.0.0.1 in any case, even if localhost and the hostname resolve to something different.

Reply:
Done